encoding/ipfix/element.go

```diff
@@ -84,154 +75,154 @@ const (
+func (fieldSpec *fieldSpecifier) ElementSpecifier() elementSpecifier {
+	return elementSpecifier{fieldSpec.ElementID, fieldSpec.EnterpriseNumber}
 }
 
-// Size returns a size of a FieldSpecifier.  This depends on Enterprise bit being set (or not).
-func (fieldSpec FieldSpecifier) Size() uint16 {
+// Size returns a size of a fieldSpecifier.  This depends on Enterprise bit being set (or not).
+func (fieldSpec fieldSpecifier) Size() uint16 {
 	if fieldSpec.E() {
 		return fieldSpecifierWEnterpriseSize
 	}
 
 	return fieldSpecifierSize
 }
 
-func (fieldSpec FieldSpecifier) String() string {
+func (fieldSpec fieldSpecifier) String() string {
 	return fieldSpec.ElementSpecifier().String()
 }
 
-// FieldDefinition describes a IPFIX field (single data).
-type FieldDefinition struct {
+// fieldDefinition describes a IPFIX field (single data).
+type fieldDefinition struct {
 	Type FieldType
 	Name string
 }
 
-func (fieldDef FieldDefinition) String() string {
+func (fieldDef fieldDefinition) String() string {
 	if fieldDef.Type == UnknownFieldType {
 		return "Unknown Field"
 	}
 
 	return fmt.Sprintf("Field '%s'", fieldDef.Name)
 }
 
-// FieldModel is a mapping of ElementSpecifier to FieldDefinition.
-type FieldModel map[ElementSpecifier]FieldDefinition
+// fieldModel is a mapping of elementSpecifier to fieldDefinition.
+type fieldModel map[elementSpecifier]fieldDefinition
 
 // Kind of immutable map (as there is no such thing as constant map in Go).
-func ipfixInformationElements() FieldModel {
-	return FieldModel{
-		ElementSpecifier{1, 0}:  FieldDefinition{Uint64, "octetDeltaCount"},
-		ElementSpecifier{2, 0}:  FieldDefinition{Uint64, "packetDeltaCount"},
-		ElementSpecifier{3, 0}:  FieldDefinition{Uint64, "deltaFlowCount"},
-		ElementSpecifier{4, 0}:  FieldDefinition{Uint8, "protocolIdentifier"},
-		ElementSpecifier{5, 0}:  FieldDefinition{Uint8, "ipClassOfService"},
-		ElementSpecifier{6, 0}:  FieldDefinition{Uint16, "tcpControlBits"},
-		ElementSpecifier{7, 0}:  FieldDefinition{Uint16, "sourceTransportPort"},
-		ElementSpecifier{8, 0}:  FieldDefinition{IPv4Address, "sourceIPv4Address"},
-		ElementSpecifier{9, 0}:  FieldDefinition{Uint8, "sourceIPv4PrefixLength"},
-		ElementSpecifier{10, 0}: FieldDefinition{Uint32, "ingressInterface"},
-		ElementSpecifier{11, 0}: FieldDefinition{Uint16, "destinationTransportPort"},
-		ElementSpecifier{12, 0}: FieldDefinition{IPv4Address, "destinationIPv4Address"},
-		ElementSpecifier{13, 0}: FieldDefinition{Uint8, "destinationIPv4PrefixLength"},
-		ElementSpecifier{14, 0}: FieldDefinition{Uint32, "egressInterface"},
-		ElementSpecifier{15, 0}: FieldDefinition{IPv4Address, "ipNextHopIPv4Address"},
+func ipfixInformationElements() fieldModel {
+	return fieldModel{
+		elementSpecifier{1, 0}:  fieldDefinition{Uint64, "octetDeltaCount"},
+		elementSpecifier{2, 0}:  fieldDefinition{Uint64, "packetDeltaCount"},
+		elementSpecifier{3, 0}:  fieldDefinition{Uint64, "deltaFlowCount"},
```

Comment:
Codacy found an issue: [avoid magic numbers like '3', create a named constant for it](https://app.codacy.com/gh/mateumann/clouseau/pullRequest?prid=6383124)